src/components/Search/Search.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import { debounce } from '../../helpers/helpers';
+
+type Props = {
+  value: string,
+  handleInputField: (event: React.ChangeEvent<HTMLInputElement>) => void
+  setHasFocusField: (val: boolean) => void
+};
+
+const Search: React.FC<Props> = ({
+  value,
+  handleInputField,
+  setHasFocusField,
+}) => {
+  const fieldref = useRef<HTMLInputElement | null>(null);
+
+  const setFieldFocus = useCallback(() => {
+    if (fieldref.current) {
+      fieldref.current.focus();
+    }
+  }, [fieldref]);
+
+  useEffect(() => {
+    setFieldFocus();
+  }, []);
```

Comment:
I don't see in the task you need to do this - even more - it looks like poor UI experience